index.html

```diff
@@ -60,17 +60,21 @@ <h3>イベント日時・場所</h3>
                     <article>
                         <h3>スピーカー</h3>
                         <div class="speaker">
+                            <!-- TODO : プロフィール画像を受け取ったら更新-->
                             <img src="images/speaker1.png" alt="" class="image"/>
```

Comment:
Got it! Let's wait until the picture is delivered.

Reply:
I got profile pics and updated it. Can you check and merge it if it looks fine?

Reply:
Yes it looks good to me!